static/actions_footer.html

```diff
@@ -0,0 +1,61 @@
+</table>
+
+<br>
+
+<table border=0 align=center>
+
+<tr>
+  <form style='text-align:center' action='add_user' method='GET'>
+    <td>
+      <input type='text' style='font-size:36;text-align:center' name='uid' autocomplete='off' value='jack'><br>
+    </td>
+    <td>
+      <input type='submit' value='Add User' style='font-size:36;text-align:center'>
+    </td>
+  </form>
+</tr>
+
+<tr>
+  <form style='text-align:center' action='add_question' method='GET'>
+    <td>
+      <input type='text' style='font-size:36;text-align:center' name='text' autocomplete='off' value='Did the chicken really?'><br>
+    </td>
+    <td>
+      <input type='submit' value='Add Question' style='font-size:36;text-align:center'>
+    </td>
+  </form>
+</tr>
+
+</table>
+
+<br>
+
+<a href='raw'><p style='font-size:24;text-align:center'>PubSub JSON Feed</p></a>
+
+<p align=center><a href='../'><font size=+3>Dashboard</font></a></p>
+
+<br>
+<br>
+
+		<div class="knsh-sticky-footer__row">
```

Comment:
The content should be wrapped in a `<div class="knsh-sticky-footer__row knsh-sticky-footer__row__m-expand"> your content here </div>` to stretch the content space so that the footer attaches to the bottom of the page.
The opening tag should be added to `actions_header.html`, the closing tag should be here above this line.